mdpow/fep.py

```diff
@@ -127,11 +127,13 @@
 import copy
 from subprocess import call
 import warnings
+import sys
 
 import numpy
 import pandas as pd
 
 import scipy.integrate
+import six
```

Comment:
Move an import from inside a function to the top:
```suggestion
import six
from six.moves import zip
```